ibm/mas_devops/roles/gencfg_jdbc/README.md

```diff
@@ -0,0 +1,45 @@
+suite_config
+============
+
+This role is used to configure db in Maximo Application Suite.
+
+Role Variables
+--------------
+
+ole Variables
+--------------
+- `db_instance_id` Defines the jdbc db instance id to be used to install the MAS app. 
```

Comment:
role variables need to define what the environment variable name is and if there is a default. See other role readmes like https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_db2wh/README.md#role-variables

Reply:
jdbc_url needs to also contain the sslConnection=true option. See https://www.ibm.com/docs/en/mas87/8.7.0?topic=administering-configuring-suite#data for what needs to be set, and so we should have something similar in the readme.

Reply:
Added them